### PR TITLE
DAT-61908 app components fix heroku site

### PR DESCRIPTION
**Pull request by crazypixel**
## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Are you adding a new package?
- [ ] yes
- [x] no

## PR Checklist
- [x] I'm using the theme colors from styled-components and checked my component in dark mode as well
- [x] Add types (Prop-types / typescript)
- [x] Commit your changes
- [x] Run `npm version **` [minor, major, patch]. Commit changes
- [x] Run `npm run lib:build`. Commit changes
- [x] Run `npm run lib:version`. Commit changes
- [x] Push changes and open a pull request

### CODE REVIEW

File: app/package.json [hunk 1]
```diff
@@ -22,8 +22,8 @@
     "react-router": "^5.1.2",
     "react-router-dom": "^5.1.2",
     "react-scripts": "3.4.1",
-    "serve": "^11.3.0",
-    "styled-components": "^5.0.0"
+    "serve": "^11.3.2",
```
Comment (Chikel) [inline app/package.json:25]: Gur suggested we lock dependencies (removing caret) - for your consideration.

Comment (crazypixel) [reply app/package.json:25]: will do!

**Pull request merged**